.devcontainer/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM mcr.microsoft.com/vscode/devcontainers/base:ubuntu
```

Comment:
Installingthe genesys cloud cli and archy.  Archy still needs a little work, but I am working on ot